site/content/docs/desktop/start/features/addons.md

```diff
@@ -22,8 +22,8 @@ Add-ons are assigned a status which may be one of:
 
 |   |         |                                                                                              |
 |---|---------|----------------------------------------------------------------------------------------------|
-|   | Release | which indicates they are of high quality and fit for purpose                                 |
-|   | Beta    | which indicates they are of reasonable quality but may be incomplete or need further testing |
+|   | Release | which indicates they are of high status and fit for purpose                                  |
+|   | Beta    | which indicates they are of reasonable status but may be incomplete or need further testing  |
```

Comment:
The spaces should be removed and the vertical line aligned.

Reply:
They were added to keep it aligned.
quality = 7
status = 6

Reply:
I guess I could drop one from the alpha line...

Reply:
The tables are generated with the minimum length.